.github/workflows/artifacts_CI.yml

```diff
@@ -20,18 +20,24 @@ concurrency:
 jobs:
   test:
     name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
-    runs-on:
-      labels: ubuntu-20.04-16core
+    runs-on: ${ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
         version:
           - "1.8"  # Version used with MVP
           - "1.9"  # Latest release
         os:
-          - ubuntu-latest
+          - ubuntu-20.04-16core
+          - macos-latest-xlarge  # M1
```

Comment:
Enabling the M1 runner in the artifacts CI ensures that the M1 artifacts are properly updated and not broken. This workflow usually only runs when the `Artifacts.toml` is changed so this is critical for ensuring a potential release is working.